Dockerfile

```diff
@@ -20,26 +20,25 @@ RUN DEBIAN_FRONTEND='noninteractive' \
 
 RUN apt-get clean
 
-# Install latest python
-ADD ./docker/install_python.sh /install_python.sh
-RUN chmod 755 /install_python.sh
-RUN /install_python.sh
-
-# Add requirements to docker
-ADD ./requirements/base.txt /requirements.txt
-RUN pip install -r /requirements.txt
-
-# Add project directory to docker
+# Add project to container
 ADD . /home/app
-RUN rm -rf /home/app/.git
-RUN chmod 755 /home/app/docker/collectstatic.sh
-RUN /home/app/docker/collectstatic.sh
-RUN  chown -R app: /home/app
 
 # Set correct environment variables.
 ENV HOME /home/app
-WORKDIR /home/app
 ENV APP_HOME /home/app
+WORKDIR /home/app
+
+RUN rm -rf /home/app/.git
+
+# Install latest python
+RUN docker/install_python.sh
```

Comment:
If this instruction appears after the "copy the code" instruction it will be executed every time there's a minor code change. This is irrelevant on platforms that do not do Docker layer caching, but otherwise not recommended. Could this be moved before the copy code part, please?

Reply:
Ok, I'll change it back to

```# Install latest python
ADD ./docker/install_python.sh /install_python.sh
RUN docker/install_python.sh
```

Reply:
:+1: :bowing_man: 
To share the context, I usually check if the Dockerfile is efficient by changing a code file unrelated to assets or dependencies and do a `docker build .`. I usually consider it good if the new build finishes under a second.